libs/fujitsu-thermal-printer/src/mocks/file_printer.ts

```diff
@@ -0,0 +1,194 @@
+import { join } from 'path';
+import { Buffer } from 'buffer';
+import {
+  existsSync,
+  lstatSync,
+  mkdirSync,
+  readFileSync,
+  readdirSync,
+  rmSync,
+  writeFileSync,
+} from 'fs';
+import { Optional, assert, err, iter, ok, sleep } from '@votingworks/basics';
+import { writeFile } from 'fs/promises';
+import {
+  FujitsuThermalPrinterInterface,
+  PrintResult,
+  PrinterStatus,
+} from '../types';
+
+export const MOCK_FUJITSU_PRINTER_STATE_FILENAME = 'state.json';
+export const MOCK_FUJITSU_PRINTER_OUTPUT_DIRNAME = 'prints';
+export const DEFAULT_MOCK_FUJITSU_PRINTER_DIR = '/tmp/mock-fujitsu-printer';
+export const DEV_MOCK_FUJITSU_PRINTER_DIR = join(
```

Comment:
Might be worth adding something to the Dev Dock readme with the basics of how to use this mock, and things like where the prints get saved to